### DISCUSSION TITLE
Are there any plans for location crafter and sublists for lists?

### DISCUSSION THREAD

**Discussion by scarfur** [jon-heard/obsidian-inline-scripts-library#6] [Shortcut-file requests]
hey there :)

as the title says. Maybe anyone already did one or both?

Sublists for lists would be mandatory for location crafter, I guess, to organize location crafters regions (in the statefile).

Greetings! :D

**Comment by jon-heard** [comment]
Hi.  I have not used the Location Crafter system myself, and I don't currently have plans to implement it.  Sublists I'm not familiar with, though it sounds like perhaps a parent list can be created with the names of other lists representing "sub" lists?

Maybe I'll look into it at some point in the future.  I have a number of other tasks on my list, though, so not right away.

**Reply by scarfur** [reply]
Thanks for your reply, no problem, I wanted to ask before I start something someone else is already writing. Yes indeed: with sublists I meant a new list type in the list plugin. Location Crafter would be quite easy to make but sublists would be mandatory. As far as I read in the plugin file, combo, file and basic types are not made for this purpose. For example there are multiple parent lists with a given name, holding more multiple parent lists with fixed names which are filled with more list elements.

**Reply by jon-heard** [reply]
Good luck with the shortcuts for the Location Crafter.  I'd be happy to answer any questions you have along the way.  If you wish to post your work here, then that'd be really neat too!

> Location Crafter would be quite easy to make but sublists would be mandatory.

After the next release (maybe early next week f I can resolve RL issues) I can get and read through the Location Crafter system and resolve sublists.  One of two things will come of that: either sublists are pretty specific to Location Crafter, or they are generally useful.  If they are specific to the Location Crafter, then I'll write shortcuts that offer sublists functionality, using the lists system.  If they are generally useful, then I'll modify the lists system to include them.
